_docs/principles/accessibility.md

```diff
@@ -3,29 +3,71 @@ title: Accessibility
+
+For example, a user who suffers from a concussion might experience difficulty processing information, so ensure that you write succinct, on-page instructions and labels. A user who experiences hand tremors as a side effect of medication has difficulty operating a mouse, so design pointer targets so they are large enough.
+
+Designing for accessibility is a best practice when using Helix because your designs will be usable by everyone regardless of whether they have any impairments.
 
-Rackspace products must be made accessible for everyone&mdash;regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
+The purpose of this document is to articulate guidelines that you can use when you design a control panel page. The list of guidelines is not exhaustive. Refer to [Web Content Accessibility Guidelines 2.1 (WCAG)](https://www.w3.org/TR/WCAG21/) for a complete list of guidelines.
+
+### Why accessibility is important
+
+Accessibility is more than just good design. It's about the empathy for and inclusion of all people. In fact, the United Nations estimates that over 1 billion people in the world live with a disability.
+
+Consider sight. Because many jobs require workers to interact with a visual display of some sort, people who experience color blindness, low sight, or no sight are particularly disadvantaged in the workplace.
+
+[The Colour Blind Awareness site](http://www.colourblindawareness.org/) estimates that:
 
 - 4.5% of the global population experience color blindness (1 in 12 men and 1 in 200 women)
 - 4% suffer from low vision (1 in 30 people)
 - 0.6% are blind (1 in 188 people)
 
-By using the following best practices, you can help ensure that your design is accessible for the visually impaired.
+With a global population of just over 7.5 billion people, 9.1% of the world's population equates to over one-half billion visually-impaired users that cannot use an inaccessible product. Inaccessible products can have a negative impact on a large percent of users.
+
+Not only can inaccessible design reflect poorly on our brand, it has the potential for legal ramifications. In 2018, there were at least 2258 lawsuits filed in federal court under Title III of the Americans with Disabilities Act (ADA), an increase of 177% of such lawsuits filed in 2017.
+
+### How Helix supports accessibility
+
+The core Helix team has taken the following steps to help you design accessible UIs:
+
+- To increase the probability that a screen reader interprets a UI correctly, we have developed components that include the appropriate `aria-*` and `role` attributes, where necessary, and use semantic markup as much as possible. These efforts alone do not guarantee screen reader accessibility, as accessibility is mostly dependent on the UI being developed.
```

Comment:
(second sentence) Accessibility isn't limited to just screen readers. Recommend...

```
These efforts alone do not guarantee accessibility, because strategies to enable accessibility are highly dependent on the UI being developed.
```

Reply:
Done.